internal/k8sutils/utils.go

```diff
@@ -94,6 +99,10 @@ func IsNodeReady(node *corev1.Node) bool {
 				// At large scale a given Node may be in an unknown state for a short period of time.
 				// However the workloads running on that Node could still be functioning.
 				if time.Since(c.LastTransitionTime.Time) < 3*time.Minute {
+					log.V(1).Info("node is in an unknown state", "node", node.Name, "lastTransitionTime", c.LastTransitionTime.Time.String())
+					if experiments.IsExperimentEnabled("NodeUnknownAsNotReady") {
```

Comment:
when experiment is enabled and node is unknown, it'd return false meaning the nodepool is down?

Reply:
correct. Current behavior is to treat it as ready, which we believe is leading to false recovery events

Additional logging is to confirm and we can selectively change the behavior as needed

Reply:
Pushed a change to tolerate X% of nodes in a pool as unknown. Renamed to "NodeUnknownAsReady" as well. Wrote a short blurb in experiements.md about how to use